google/cloud/storage/bucket_metadata.cc

```diff
@@ -76,6 +83,21 @@ std::ostream& operator<<(std::ostream& os, CorsEntry const& rhs) {
             << join(", ", rhs.response_header) << "]}";
 }
 
+std::ostream& operator<<(std::ostream& os, BucketOnlyPolicy const& rhs) {
+  return os << "BucketOnlyPolicy={enabled=" << std::boolalpha
```

Comment:
This just occurred to me, although it's already present elsewhere -- if we're going to be setting flags on the passed in stream, should we be resetting it afterward? E.g.

    std::ios_base::fmtflags oldflags = os.setf(std::ios_base::boolalpha);
    // Do stuff
    os.flags(oldflags);
    return os;